docs/competition/event/README.md

```diff
@@ -1,25 +0,0 @@
----
-original:
-  authors: Fiona Gillan, Richard Barlow, Rob Gilton and Thomas Leese
-  url: https://bitbucket.org/rspanton/sr-event-coord/wiki/Home
----
-# Competition Event Documentation
-
-This is the Competition Event Coordinator's operations documentation.
-
-## Subordinate Roles
-
-Role | Volunteer | Documentation
------|-----------|--------------
-Reward Manager |
-Competition Volunteer Welfare Coordinator |  | [Docs](https://gist.github.com/howiegoing/5876963bc0dcb63a6fa1028b02d1a5a6)
```

Comment:
This item doesn't seem to be covered by another section. Worth importing / rewriting internally?

Reply:
I believe this is superseded by https://studentrobotics.org/runbook/competition/volunteers/refreshments/. Is there content you feel is in the gist which is missing from that page?
(Note that our approach seems to have since changed from occasionally taking refreshments around to stationary posts to essentially always having roving refreshments)

Reply:
That page looks sufficient to me!